lib/Common/Memory/RecyclerWriteBarrierManager.cpp

```diff
@@ -66,6 +66,16 @@ X64WriteBarrierCardTableManager::OnThreadInit()
     ::GetCurrentThreadStackLimits(&stackEnd, &stackBase);
 #endif
 
+#ifdef X64_WB_DIAG
+    this->_stackbase = (char*)stackBase;
+    this->_stacklimit = (char*)stackEnd;
+#endif
+
+    // on Windows server 2012 stack limit can expand with process running, and causes
```

Comment:
>Windows server 2012 [](start = 10, length = 19)

Is growing stack limit only a feature for Windows server 2012, ant not for all others?